src/modules/OTA-api/services/open-terms-archive.ts

```diff
@@ -0,0 +1,31 @@
+import fetch, { launchHeadlessBrowser, stopHeadlessBrowser } from 'open-terms-archive/fetch';
+import filter from 'open-terms-archive/filter';
+
+export const getVersion = async (documentDeclaration: any, config: any) => {
+  await launchHeadlessBrowser();
+  const { content, mimeType } = await fetch({
+    url: documentDeclaration.fetch,
+    executeClientScripts: documentDeclaration.executeClientScripts,
+    cssSelectors: documentDeclaration.select,
+    config,
+  });
+  await stopHeadlessBrowser();
+
+  const mdContent = await filter({
+    content,
+    mimeType,
+    documentDeclaration: {
+      location: documentDeclaration.fetch,
+      contentSelectors: documentDeclaration.select,
+      noiseSelectors: documentDeclaration.remove,
```

Comment:
Our API is not really inconsistent, is just that there is a difference between the document declaration and the representation in the code, because the purpose is not the same and each have terms which seems more appropriate to the context.

Our API is not really inconsistent, it's just that there is a difference between the syntax of user document declaration and the document declaration representation in the code, as the purpose is not the same and each has terms that seem more appropriate to their context. However, they should probably be named differently to avoid any ambiguity.